Persephone/Controllers/AlbumDetailView.swift

```diff
@@ -47,16 +47,32 @@ class AlbumDetailView: NSViewController {
 
   func getAlbumSongs(for album: Album) {
     App.mpdClient.getAlbumSongs(for: album.mpdAlbum) { (mpdSongs: [MPDClient.MPDSong]) in
-      self.dataSource.albumTracks = mpdSongs.map {
-        return Song(mpdSong: $0)
-      }
+      self.dataSource.setAlbumSongs(
+        mpdSongs.map { Song(mpdSong: $0) }
+      )
+
+      self.getBigCoverArt(song: self.dataSource.albumSongs.first!.song ?? self.dataSource.albumSongs[1].song!)
 
       DispatchQueue.main.async {
         self.albumTracksView.reloadData()
       }
     }
   }
 
+  func getBigCoverArt(song: Song) {
+    let coverArtService = CoverArtService(song: song)
+
+    coverArtService.fetchBigCoverArt()
+      .done() { image in
```

Comment:
Does `done()` have an optional parameter to switch queues?

Reply:
It does!